### ISSUE TITLE
ISAM create sysaccounts question

### ISSUE THREAD

**Issue by cookieninja** [IBM-Security/isam-ansible-collection#126]
I'm having trouble creating an ISAM sysaccount that is assigned to groups. I can see there is a parameter but cannot see an example showing the expected format. Can someone show me an example please?

**Comment by tombosmansibm** [comment]
using this role (/base/configure_sysaccounts) , adding a user that is member of an EXISTING group would be like this.

sysaccounts:
        - sysaccount_action: create
          id: sample_user
          password: "{{ sample_user_password }}" # password should be secured with ansible-vault
          groups:
             - samplegroup1
             - samplegroup2

However, I do see that there's code missing in the modules to modify groupmemberships and the "modify" function for a user only changes the password.
Note that it may better to use "management authorization" with LDAP users/groups instead. This allows you to manage the access in an identity management system you probably already have, and allows your admin users to login with an id that they can manage themselves (in LDAP).

**Comment by cookieninja** [comment]
Thanks! I had been trying to use create_sysaccount but the formatting used when providing a list of groups seems to be the same

Not sure if it's my understanding of your reply or what you wrote, but I figured it out whilst trying to make that role work that I needed to list the groups like this. I was surprised to see I could list two different groups under the same heading in the way I did too, but it works so not complaining!

groups:
    - id: "<group-name1>"
    - id: "<group-name2>"

Although I've used ISAM (v5.1+) for some time, I'm new to ansible and relatively inexperienced with YAML so it's probably just my understanding :( Probably going to get sick of me asking questions until I get familiar enough with YAML and the docs to figure everything out myself :-s
